include/valijson/validation_visitor.hpp

```diff
@@ -347,6 +347,82 @@ class ValidationVisitor: public constraints::ConstraintVisitor
         return numValidated > 0;
     }
 
+    /**
+    * @brief    Validate current node against a FormatConstraint
+    * 
+    * @param    constraint  Constraint that the target must validate against
+    * 
+    * @return   \c true if validation succeeds; \c false otherwise
+    */
+    bool visit(const FormatConstraint &constraint) override
+    {
+        const std::string s = m_target.asString();
+        const std::string format = constraint.getFormat();
+        if (format == "date")
+        {
```

Comment:
Just being pedantic, but the convention in this project is to open the `{` on the same line as the condition.

Reply:
Fixed that